tests/PointTest.php

```diff
@@ -255,4 +256,48 @@ public function providerToArrayAndInterfaces() : array
             ['POINT ZM (4.5 5.6 6.7 7.8)', [4.5, 5.6, 6.7, 7.8]],
         ];
     }
+
+    /**
+     * @dataProvider providerAzimuth
+     *
+     * @param string $observerWkt     The WKT of the point, representing the observer location.
+     * @param string $subjectWkt      The WKT of the point, representing the subject location.
+     * @param array  $azimuthExpected The expected azimuth.
+     *
+     * @return void
+     */
+    public function testAzimuth(string $observerWkt, string $subjectWkt, $azimuthExpected): void
+    {
+        $this->requiresGeometryEngine();
+
+        if ($this->isGEOS() || $this->isMySQL() || $this->isMariaDB() || $this->isSpatiaLite()) {
```

Comment:
Should this be `if (! $this->isPostGIS())`?

Reply:
Absolutely! Fixed!